index.js

```diff
@@ -29,7 +29,7 @@ function multi(keys, dep) {
 }
 
 module.exports = function (str) {
-	return str.replace(/import (.*) from (.*)/gi, (_, req, dep) => {
+	return str.replace(/import ([\s\S]*?) from (['"][^'"]+['"]);?/gi, (_, req, dep) => {
```

Comment:
I _think_ we'd only need the first change, right? The rest of the helper functions work alongside the more vague `from (.*)` segment.

Was there a particular reason you added this second half?

Reply:
That's to support multiple statements, otherwise `.*` would catch a part of the next import statement.

Reply:
And if we change it to `['"]([^'"]+)['"]` we could remove `clean` function from some (or even all) places.

Reply:
I've updated the branch — let me know what you think!